dpdata/plugins/gaussian.py

```diff
@@ -15,6 +19,19 @@
     from dpdata.utils import FileType
 
 
+def register_hessian_data(data):
+    if "hessian" in data:
+        dt = DataType(
+            "hessian",
+            np.ndarray,
+            (Axis.NFRAMES, -1, -1),
```

Comment:
Why using -1? The shape should be `(Axis.NFRAMES, Axis.NATOMS, 3, Axis.NATOMS, 3)`.